tests/test_group.rs

```diff
@@ -312,106 +312,344 @@ fn group_operations() {
             message_alice,
             mls_plaintext_bob.as_application_message().unwrap()
         );
-    }
-}
-/*
-    // Bob updates and commits
-    let update_proposal_bob = group_bob.create_update_proposal(None);
-    let (commit2, ms2, _) = group_bob.create_commit(None);
-
-    group_alice_1234.process_proposal(update_proposal_bob);
-    group_alice_1234.process_commit(commit2.clone());
-    group_bob.process_commit(commit2);
-
-    group_alice_1234.tree.print(&format!("\n{:?}", ms2));
-
-    // Alice updates and commits
-    let update_proposal_alice = group_alice_1234.create_update_proposal(None);
-    let (commit3, ms3, _) = group_alice_1234.create_commit(None);
 
-    group_bob.process_proposal(update_proposal_alice);
-    group_alice_1234.process_commit(commit3.clone());
-    group_bob.process_commit(commit3);
-
-    group_alice_1234.tree.print(&format!("\n{:?}", ms3));
-
-    // Alice updates and Bob commits
-    let update_proposal_alice = group_alice_1234.create_update_proposal(None);
-    group_bob.process_proposal(update_proposal_alice);
-    let (commit4, ms4, _) = group_bob.create_commit(None);
-
-    group_bob.process_commit(commit4.clone());
-    group_alice_1234.process_commit(commit4);
-
-    group_alice_1234.tree.print(&format!("\n{:?}", ms4));
+        // === Bob updates and commits ===
+        let bob_update_key_package_bundle = KeyPackageBundle::new(
+            &[ciphersuite_name],
+            &bob_credential_bundle,
+            mandatory_extensions.clone(),
+        );
 
-    // Bob updates and Alice commits
-    let update_proposal_bob = group_bob.create_update_proposal(None);
-    group_alice_1234.process_proposal(update_proposal_bob);
-    let (commit5, ms5, _) = group_alice_1234.create_commit(None);
+        let update_proposal_bob = group_bob.create_update_proposal(
+            &[],
+            &bob_credential_bundle,
+            bob_update_key_package_bundle.get_key_package().clone(),
+        );
+        let (mls_plaintext_commit, _, kpb_option) = match group_bob.create_commit(
```

Comment:
Should we check what's in the `Welcome` as well even if we don't use it? (Same below.)

Reply:
Yes, that check was missing.